### PR TITLE
Add 'caseSensitive' option

### PR DESCRIPTION
**Pull request by kevipapo1**
If 'caseSensitive' is true, strip_html() will not convert input to lowercase, allowing for case-sensitive queries.

Implementation, README, and example are all added.

### CODE REVIEW

File: src/jquery.quicksearch.js [hunk 1]
```diff
@@ -331,4 +335,4 @@
 		
 	};
 
-}(window.jQuery, this, document));
+}(window.jQuery, this, document));
```
Comment (nitriques) [inline src/jquery.quicksearch.js]: Could we keep the empty line at the end of file please ?

**Comment by nitriques** [comment]
Great! Simple little thing missing and we are ready to merge!

**Comment by kevipapo1** [comment]
Should be good now.

**Comment by nitriques** [comment]
Awesome!

**Comment by nitriques** [comment]
Manually merged and squashed as d24c63c

**Comment by nitriques** [comment]
Released in https://github.com/DeuxHuitHuit/quicksearch/releases/tag/2.3.0

**Comment by nitriques** [comment]
it's up on npm and bower too. Thanks again!

**Comment by nitriques** [comment]
Examples have been updated too: https://deuxhuithuit.github.io/quicksearch/r/examples/

Tell me if I missed anything!

**Comment by kevipapo1** [comment]
Everything looks good. Package installs successfully via npm install. Thanks for the pull!

**Comment by nitriques** [comment]
My pleasure, thanks for contributing!

**Pull request closed without merging**